Dockerfile

```diff
@@ -10,3 +9,4 @@ RUN npm ci
 RUN npm i -g grunt-cli
 
 COPY . .
+CMD ["node", "index.js"]
```

Comment:
Without this it doesn't run in local and doesn't run test.

Reply:
How did this work in the past?

Reply:
After fixing the build in node 8, 10 and 12 in another PR, the original version of this file is working again.